pkg/reconciler/common/releases.go

```diff
@@ -181,7 +181,16 @@ func AppendManifest(manifest *mf.Manifest, yamlLocation string) error {
 	}); err != nil {
 		return err
 	}
+
 	for i := range files {
+		folderInfo, err := os.Stat(files[i])
+		if os.IsNotExist(err) {
+			return fmt.Errorf("failed to find file/dir at path: %s: %s ", files[i], err.Error())
+		}
```

Comment:
to me it looks like`IsNotExist` is the one error which is not guranteed to happen here as the file list is populated by the above walk operation.

so shall we either return for all error here. It looks like if the error is of any kind it will be ignored.

if this is to add whether the a file exists when a single file path is provided, then shall we move this check to the beginning of the AppendManifest function. 

In addition, wouldn't the `filepath.Walk` catch that error if the `yamlLocation` param is a non-existent file.